src/views/wiki/edition/OutingEditionView.vue

```diff
@@ -57,7 +57,7 @@
             <document-title :document="route" />,
             <document-rating :document="route" />
           </input-checkbox>
-          <document-link :document="route" target="_blank">
+          <document-link :document="route" target="_blank" :title="$gettext('Open in a new tab')">
```

Comment:
Depending on the browser configuration, I do think it can be a new tab or a new window (although few people use the latter option, I think)